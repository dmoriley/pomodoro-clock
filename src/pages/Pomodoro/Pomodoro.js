import React from 'react';
// import PropTypes from 'prop-types';
import styles from './Pomodoro.module.scss';

const Pomodoro = props => {
  return (
    <div className={styles.root}>
      <div className={styles.box}>
        Pomodoro
      </div>
    </div>
  );
};

// Pomodoro.defaultProps = {

// };

// Pomodoro.propTypes = {

// };

export default Pomodoro;