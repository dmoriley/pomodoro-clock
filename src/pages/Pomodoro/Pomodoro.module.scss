.root {
  min-height: 100vh;
  min-width: 100vw;
  background-color: var(--dark-grey-blue);
  // background: linear-gradient(rgb(102, 215, 250), $main-colour);
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
}

.box {
  color: fade-out( white, .15);
  padding: var(--s-size) var(--s-size);
  text-align:center;
  width: 50rem;
  margin-bottom: 2.0rem;
}

h1 {
  font-size: var(--xl-size);
}

footer {
  p{
    color: var(--rose-red);
    font-family: monospace;
    font-size: var(--s-size);
    margin: 0;

    span {
      color: var(--blue);
    }
  }
}

// @media only screen and (min-width: $desktop-breakpoint) {
// // desktop css in here
//   .box-layout__box {
//     // width: auto;
//   }
// }