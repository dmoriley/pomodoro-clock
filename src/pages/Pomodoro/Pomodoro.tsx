import React from 'react';
import styles from './Pomodoro.module.scss';
import { ClockContainer } from '../../components/index';

const Pomodoro: React.FC = () => {
  return (
    <div className={styles.root}>
      <section className={styles.box}>
        <header>
          <h1>Pomodoro Clock</h1>
        </header>
        <ClockContainer />
        <footer>
          <p>
            Designed and Coded by
            <br />
            <span>David O'Riley</span>
          </p>
        </footer>
      </section>
    </div>
  );
};

export default Pomodoro;
