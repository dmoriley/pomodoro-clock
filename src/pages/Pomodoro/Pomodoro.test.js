import React from 'react';
import Pomodoro from './Pomodoro';

describe('Pomodoro', () => {
   it('renders without error', () => {

   });
});