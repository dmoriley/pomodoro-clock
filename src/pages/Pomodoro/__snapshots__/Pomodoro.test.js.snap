// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pomodoro renders without error 1`] = `
<div
  class="root"
>
  <section
    class="box"
  >
    <header>
      <h1>
        Pomodoro Clock
      </h1>
    </header>
    <div
      class="root"
    >
      <div>
        <div>
          break length
        </div>
        <div>
          session length
        </div>
      </div>
      <div>
        clock
      </div>
      <div>
        buttons
      </div>
    </div>
    <footer>
      <p>
        Designed and Coded by
        <br />
        <span>
          David O'Riley
        </span>
      </p>
    </footer>
  </section>
</div>
`;
